AlgebraInLean/Chapter01/Sheet02.lean

```diff
@@ -4,10 +4,11 @@ namespace AlgebraInLean
 
 /-
 Groups aren't the only important structures in algebra. By including or excluding certain properties
-from our definition, we create different structures.
+from our definition, we can create different structures.
 -/
 
-/-- A magma is the simples algebraic structure, which only has a closed binary operation -/
+/-- A magma is the simplest algebraic structure. It is a set along with a binary operation with no
+  additional properties imposed -/
```

Comment:
Multiline comments should have the comment delimiters on their own lines. I think specifying "closed" is useful this early on. I'd also consider changing the preposition "with" since it's repetitive, but I'm not sure whether there are any good options (maybe "[] along with a closed binary operation but no additional imposed properties"? Seems a bit stilted).